rules/windows/process_creation/proc_creation_win_fltmc_unload_driver_sysmon.yml

```
@@ -1,12 +1,12 @@
-title: Sysmon Driver Unload
+title: Sysmon Filter Driver Unloaded Via Fltmc.EXE
 id: 4d7cda18-1b12-4e52-b45c-d28653210df8
```

```suggestion
id: 4d7cda18-1b12-4e52-b45c-d28653210df8
related:
    - id: 4931188c-178e-4ee7-a348-39e8a7a56821 # Generic
      type: similar
```
Will get the 2 alerts